frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/helpers.ts

```diff
@@ -0,0 +1,11 @@
+export const getAvailableCDName = (cds) => {
+  let index = 1;
+  let name = `cd-drive-${index}`;
+  cds.forEach((cd) => {
```

Comment:
this is what happens here though, i'm finding the first free index.
the cd's names doesn't sync with the amount of the cds

Reply:
this only works because there are two CDs at maximum. Suppose you have two cds

`['cd-drive-2', 'cd-drive-1']`

you will not find `cd-drive-3` with this approach.

 The code should be correct regardless the usage.

Reply:
works for me with mock array, tried both cases of needing the last index and a case where i'm using a free index that is smaller than the last one

![image](https://user-images.githubusercontent.com/24938324/70907017-6ab47b80-2010-11ea-844e-aad372005f08.png)
<img width="348" alt="Screen Shot 2019-12-16 at 14 28 22" src="https://user-images.githubusercontent.com/24938324/70907031-72742000-2010-11ea-8559-fa1be0ef1201.png">

Reply:
Doesn't work with the example I gave.

![Screenshot from 2019-12-16 14-17-07](https://user-images.githubusercontent.com/3648838/70909886-c3831480-200e-11ea-9430-cb2d60f61d9f.png)